quattor/functions/filesystem.pan

```diff
@@ -49,6 +49,32 @@ function lvs_add = {
     SELF;
 };
 
+@documentation{
+    desc = Adds a list of logical volumes to a volume group with mutiple lv options. To replace function lvs_add which only sets size.
+    arg = volume group, list of logical groups long with optons.  size and raid type are set if defined.
+}
+function lvo_add = {
+    function_name = 'lvo_add';
+    if (length (ARGV) != 2) {
+        error ("%s: should get 2 arguments", FUNCTION);
+    };
+    vg = ARGV[0];
+    lv = ARGV[1];
```

Comment:
This is a list so it should be lvs (which is horrible) so `l_vols` ? or `lv_list` ?